docs/RELEASE.md

```diff
@@ -0,0 +1,75 @@
+# Docker Machine Release Process
+
+The Docker Machine release process is fairly straightforward and as many steps
+have been taken as possible to make it automated, but there is a procedure and
+several "checklist items" which should be documented.  This document is intended
+to cover the current Docker Machine release process.  It is written for Docker
+Machine core maintainers who might find themselves performing a release.
+
+1.  **Version Bump** -- When all commits for the release have been merged into
+master and/or the release branch, submit a pull request bumping the `Version`
+variable in `version/version.go` to the release version.  Merge said pull
+request.  
+2. **Compile Binaries** -- Pull down the latest changes to master and
+cross-compile the core binary and plugin binaries for each supported OS /
+architecture combination.  As you may notice, this can potentially take a _very_
+long time so you may want to use a script like [this one for cross-compiling
+them on DigitalOcean in
+parallel](https://gist.github.com/nathanleclaire/7f62fc5aa3df19a50f4e).
```

Comment:
Er, never mind - I see now from the original description that you're planning on doing some follow-ups :grin: